pkg/injector/test_fixtures/expected_envoy_bootstrap_config.yaml

```diff
@@ -23,7 +23,7 @@ dynamic_resources:
     resource_api_version: V3
 static_resources:
   clusters:
-  - connect_timeout: 0.25s
+  - connect_timeout: 0.250s
```

Comment:
Was this change intentional?

Reply:
When the xds structs are marshaled to YAML the 0 after .25 remains

Reply:
My bad, this looks good!